modules/ztp-setting-managed-bare-metal-host-kernel-parameters-manual-workflow_gitops_workflow.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+[id="setting-managed-bare-metal-host-kernel-parameters_{context}"]
+= Setting managed bare-metal host kernel parameters
+
+Configure kernel parameters for managed bare-metal hosts to customize the boot process. This is useful for clusters with specific environmental requirements. For example, configure the `rd.net.timeout.carrier` kernel parameter to facilitate static networking or to receive a DHCP address before downloading the root file system. 
+
+[NOTE]
+====
+In {product-title} {product-version}, you can only add kernel parameters. You can not replace or delete kernel parameters.
+====
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have logged in to the hub cluster as a user with cluster-admin privileges.
+
+.Procedure
+
+. Create the `InfraEnv` CR and edit the `spec.kernelArguments` specification to configure kernel parameters.
+
+.. Save the following YAML in an `InfraEnv-example.yaml` file, replacing the variables to match the details of your cluster:
++
+[source,yaml,options="nowrap",role="white-space-pre"]
+----
+apiVersion: agent-install.openshift.io/v1beta1
+kind: InfraEnv
+metadata:
+  annotations:
+    argocd.argoproj.io/sync-wave: "1"
+  name: <cluster_name>
+  namespace: <cluster_name>
+spec:
+  clusterRef:
+    name: <cluster_name>
+    namespace: <cluster_name>
+  pullSecretRef:
+    name: pull-secret
+  kernelArguments:
+    - operation: append <1>
+      value: audit=0 <2>
+    - operation: append 
+      value: trace=1
+  sshAuthorizedKey: <public_key> <3> 
+    proxy: 
+      httpProxy: <proxy_address> 
+      httpsProxy: <proxy_address> 
+  ignitionConfigOverride: <ignition_config> <4>
+  nmStateConfigLabelSelector:
+    matchLabels:
+      nmstate-label: <cluster_name>
+   additionalNTPSources: 
+     - 1111:2222:3333:4444::2 
```

Comment:
This file needs to contain the template syntax used by the sitegen tool. The fields will be automatically substituted when the SiteConfig CR triggers generation of this (and other) installation CR. Here is the current version with the additional kernelArguments added:
```
apiVersion: agent-install.openshift.io/v1beta1
kind: InfraEnv
metadata:
  annotations:
    argocd.argoproj.io/sync-wave: "1"
  name: "{{ .Cluster.ClusterName }}"
  namespace: "{{ .Cluster.ClusterName }}"
spec:
  clusterRef:
    name: "{{ .Cluster.ClusterName }}"
    namespace: "{{ .Cluster.ClusterName }}"
  kernelArguments:
    - operation: append <1>
      value: audit=0 <2>
    - operation: append 
      value: trace=1
  sshAuthorizedKey: "{{ .Site.SshPublicKey }}"
  proxy: "{{ .Cluster.ProxySettings }}"
  pullSecretRef:
    name: "{{ .Site.PullSecretRef.Name }}"
  ignitionConfigOverride: "{{ .Cluster.IgnitionConfigOverride }}"
  nmStateConfigLabelSelector:
    matchLabels:
      nmstate-label: "{{ .Cluster.ClusterName }}"
  additionalNTPSources: "{{ .Cluster.AdditionalNTPSources }}"
```

Reply:
Appreciate the review @imiller0. I've implemented your feedback. Does it look good to you?